bedrock/security/templates/security/web-bug-bounty.html

```diff
@@ -134,6 +141,7 @@ <h2 id="exclusions">Exclusions</h2>
     <li>Source code disclosures, as most of our code is open source</li>
     <li>Vulnerabilities discovered shortly after their public release</li>
     <li>Outdated TLS configurations which remain to support downloads from Windows XP systems</li>
+    <li>SSRF reports on services that are designed to load resources from the internet</li>
```

Comment:
services that load things from the internet _might_ also have a legit SSRF problem, you just can't prove it using "Blind SSRF" probes. Is there a clearer way to word this? Maybe just that:

> "Blind SSRF" reports on services that [etc]

Reply:
yes indeed, I can add that, thanks